src/Console/Migrate.php

```diff
@@ -97,6 +209,36 @@ protected function target(): string
         return $this->validatedOption('schema-to');
     }
 
+    protected function tableNames(): array
+    {
+        return $this->option('tables');
+    }
+
+    protected function excludeTables(): array
+    {
+        return $this->option('exclude-tables');
+    }
+
+    protected function getMigrationOption()
+    {
+        return $this->choice('Please choose option to run migration on which connection?', self::$choices, 0);
+    }
+
+    protected function resolveTableListOption()
```

Comment:
1. Add method return typing: `: bool`
2. The method for requesting information from the user cannot begin with the word "resolve". You need to use "get".